data/timeliner.yaml

```diff
@@ -847,6 +847,14 @@ attribute_mappings:
   description: 'Last Scavenge Time'
 place_holder_event: false
 ---
+data_type: 'msie:cookie:entry'
```

Comment:
Let's make this `msie:webcache:cookies` to align with the other MSIE webcache data types